### ISSUE TITLE
Support React v15.0 and later

### ISSUE THREAD

**Issue by kpaxqin** [sanniassin/react-input-mask#39]
Currently I'm trying to update my react version to 15.0.2 but blocked my this lib(now the peerDependency is react >=0.12).
Do we have a plan to support the newest version?

**Comment by sanniassin** [comment]
@kpaxqin It's likely another module incompatibility:
https://github.com/sanniassin/react-input-mask/issues/35

**Comment by kpaxqin** [comment]
>npm ERR! peerinvalid The package react@15.0.2 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer react-input-mask@0.6.4 wants react@>=0.12.0

Error message here @sanniassin

**Comment by sanniassin** [comment]
@kpaxqin Try to delete your node_modules dir and run npm install again. If error persists, please provide your package.json and npm version.

**Comment by sanniassin** [comment]
@kpaxqin Still can't reproduce even with npm2. Can you make and post on github a minimal project which fails on npm install?

**Comment by kpaxqin** [comment]
@sanniassin I'm trying to update my npm version and see what happened but meet some problem with the network to npm server, and if still can't resolve it I will try to post a minimal project. Thanks.

**Comment by kpaxqin** [comment]
@sanniassin I think I found that problem and it's caused by the react-router@0.13

When I have react-router@0.13 as a dependency and try to update to react@15, it will fail with the error message I posted before (which says the validation also failed becuz of react-input-mask), but when remove react-router, it can update successfully. [This minimal project ](https://github.com/kpaxqin/test-react-update) shows how to reproduce it.

So it should not be a defect with react-input-mask. 

Thanks for your support.
